pyproject.toml

```diff
@@ -39,6 +39,11 @@ jsonschema = "^4.21.1"
 brick-tq-shacl = {optional = true, version="0.3.2a4"}
 werkzeug="^2.3.7"
 types-jsonschema = "^4.21.0.20240311"
+abbreviations = "^0.2.5"
+langchain = "^0.2.3"
+langchain-community = "^0.2.4"
+pyenchant = "^3.2.2"
+scikit-learn = "^1.5.0"
```

Comment:
Can we gate your dependencies behind a feature flag? Call it something like `label_parsing`

Reply:
The latest commit also addresses this, please let me know if this is resolved.